application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -84,9 +84,11 @@
 
     private static final List<OptionData> inviteOptions = List.of(
             new OptionData(OptionType.STRING, MAX_USES_OPTION,
-                    "The amount of times the invite can be used, default is infinity", false),
+                    "The amount of times the invite can be used, default is infinity", false)
+                        .setRequiredRange(0, 100),
             new OptionData(OptionType.INTEGER, MAX_AGE_OPTION,
-                    "Max age in seconds. Set this to 0 to never expire, default is 1 day", false));
+                    "Max age in seconds. Set this to 0 to never expire, default is 1 day", false)
+                        .setRequiredRange(0, 604800));
```

Comment:
This number is an unreadable mess. Use the `java.time` API to make it clear what this is.

Reply:
Changed it to use java.time.Duration